### DISCUSSION TITLE
Make /follow [nameofcharacter] break follow.

### DISCUSSION THREAD

**Discussion by Aubergin** [chromiecraft/chromiecraft#3266] [Suggestions]
In Live Classic and Retail realms using the command /Follow (otherplayer) can be broken by issuing command /follow [nameofYourcharacter].

The only way to break follow now is to make a move command. This breaks channeling or spellcasts.

I dont know how easy this change is but it would help me a ton!
~Aubergin

**Comment by 55Honey** [comment]
Hello Aubergin,
Thanks for your suggestion.

What you're asking for is not related to the core. We can't make it happen server side. Follow is a client side function.

Try this in a macro:
`/run FollowUnit("player")`
It is supposed to work, according to https://wowwiki-archive.fandom.com/wiki/API_FollowUnit

**Comment by Aubergin** [comment]
Sadly, it does not work.

**Comment by 55Honey** [comment]
Ah sorry, my bad. I forgot to go back in history for that page:
https://wowwiki-archive.fandom.com/wiki/API_FollowUnit?oldid=925946

The old revision says it's not possible.
